frontend/src/components/dialogs/GDialog.vue

```diff
@@ -18,11 +18,13 @@ SPDX-License-Identifier: Apache-2.0
           </template>
         </v-toolbar-title>
       </v-toolbar>
-      <v-card-text class="pa-3" :style="{'max-height': maxHeight}" ref="contentCard">
+      <slot name="tabbar"></slot>
+      <v-card-text :class="{ 'pa-3' : !disableContentInset, 'pa-0' : disableContentInset }" :style="{ 'max-height': maxHeight }" ref="contentCard">
```

Comment:
Why do we use `v-card-text` if we don't want the padding. Here we we the slot tabbar. We do not have this in gPopper. Why is the name of this slot `tabbar`. This is just one way we could use this slot.

Reply:
I added a generic slot instead of `tabbar` and removed the `disableContentInset` prop